src/components/shared/epic.tsx

```diff
@@ -0,0 +1,124 @@
+// ----- Imports ----- //
+import { css, SerializedStyles } from '@emotion/core';
+import { basePx, headlineFont } from 'styles';
+import { from } from '@guardian/src-foundations/mq';
+import { palette, brandAlt } from '@guardian/src-foundations';
+import React, { useState, useEffect, useRef } from 'react';
+import { nativeClient } from 'native/nativeApi';
+import { SvgArrowRightStraight } from "@guardian/src-svgs"
+import { ThemeProvider } from 'emotion-theming'
+import { Button, buttonReaderRevenue } from '@guardian/src-button'
+
+
+// ----- Styles ----- //
+
+const EpicStyles = (): SerializedStyles => css`
+        width: calc(100% - ${basePx(4)});
+        margin: ${basePx(1)};
+
+        ${from.wide} {
+            margin: ${basePx(1, 0)};
+        }
+
+        clear: both;
+
+        border-top: 1px solid ${brandAlt[400]};
+        background: ${palette.neutral[97]};
+        padding: ${basePx(1)};
+        font-family: 'Guardian Text Egyptian Web';
+        clear: left;
+
+        h1:first-of-type {
+            ${headlineFont}
```

Comment:
```suggestion
            ${headline.large()}
```
Not sure which is the right size here, the list of options is [in the docs](https://www.theguardian.design/2a1e5182b/p/930d69-typography/t/735a67).